test/integration/smart_answer_flows/benefit_cap_calculator_test.rb

```diff
@@ -11,6 +11,12 @@ class BenefitCapCalculatorTest < ActiveSupport::TestCase
   end
 
   context "Benefit cap calculator" do
+    setup do
+      WebMock.stub_request(:get, "#{Plek.new.find('imminence')}/areas/IG6%202BA.json").
+        to_return(body: File.open(fixture_file('imminence/london.json')))
+      WebMock.stub_request(:get, "#{Plek.new.find('imminence')}/areas/B1%201PW.json").
+        to_return(body: File.open(fixture_file('imminence/national.json')))
```

Comment:
I think you should be using `imminence_has_areas_for_postcode` here instead of using `WebMock`. Unless you've discounted it; in which case you should mention why in the commit note.

You'll need something like:

```
require 'gds_api/test_helpers/imminence'

include GdsApi::TestHelpers::Imminence

imminence_has_areas_for_postcode("IG6%202BA",  [{ slug: 'london', country_name: 'England' }])
imminence_has_areas_for_postcode("B1%201PW",   [{ slug: "birmingham-city-council", country_name: 'England' }])
```